src/jalapeno-pickled-eggs.md

```diff
@@ -0,0 +1,32 @@
+# Jalapeno Pickled Eggs
+
+![eggs fermenting in jar](pix/jalapeno-eggs.webp)
+
+Pickled eggs with a spicy twist.
+
+- Hard boiling: 25 min
+- Pickling: 20 min
+- Fermentation: 1-2 weeks
+
+## Ingredients
+- 9-10 hard boiled eggs ([recipe here](https://based.cooking/eggs))
+- 2 3/4 cups white vinegar
+- 3/4 cups water
+- 1 tbsp salt
+- 5 jalapenos sliced
+- 1/4 white onion sliced
+- 1-2 garlic cloves
+- Ziploc bag (optional)
+
+## Directions
+1. Put eggs in jars with jalapenos, onions, and garlic cloves between eggs.
+2. Pour water and vinegar into saucepan, add salt, heat, and mix until dissolved.
+3. Pour brine into jar, leaving 1/4" headspace (using a [jar funnel](https://www.ebay.com/sch/i.html?_from=R40&_trksid=p2380057.m570.l1313&_nkw=jar+funnel&_sacat=0) helps). Submerge all ingredients completely with brine, as unsoaked food will rot. This can be insured by cramming a partially inflated ziploc bag underneath the lid, raising the water level.
```

Comment:
```suggestion
3. Pour brine into jar, leaving 1/4" headspace (using a [jar funnel](https://www.ebay.com/sch/i.html?_from=R40&_trksid=p2380057.m570.l1313&_nkw=jar+funnel&_sacat=0) helps). Submerge all ingredients completely with brine, as unsoaked food will rot. This can be ensured by cramming a partially inflated ziploc bag underneath the lid, raising the water level.
```